spec/src/main/asciidoc/configprofile.asciidoc

```diff
@@ -0,0 +1,77 @@
+//
+// Copyright (c) 2020 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// Contributors:
+// Emily Jiang
+
+
+[[configprofile]]
+== Config Profile
+
+Config Profile indicates the project phase, such as dev, testing, live, etc. 
+
+=== Specify Config Profile
```

Comment:
Why not use the term which is established in JavaEE: ProjectStage ?

This also has the benefit, that there is always a ProjectStage. Even if you don't specify any, you'll fallback to Production. This allows the following neat use case:

> myproject.security.enabled.Production=true
> myproject.security.enabled=false

Which means: enable security in production, but have it disabled anywhere else.

Reply:
In my opinion, `profile` seems to be a more "popular" term to specify the configuration. It is used by Spring for instance.

Is there any particular reason why EE went with `ProjectStage` instead of `Profile`?

Reply:
I think I will stick with `profile` for now.

Reply:
I will keep `profile` for now.